R/vcr_test_path.R

```diff
@@ -0,0 +1,31 @@
+#' Locate file in tests directory
+#'
+#' This function, similar to `testthat::test_path()`, is designed to work both
+#' interactively and during tests, locating files in the `tests/` directory.
+#'
+#' @param ...	Character vectors giving path component.
```

Comment:
maybe do a bit better in explaining this parameter than testthat does - i.e,. each character string gets added on to the path, e.g., `vcr_test_path("a", "b")` becomes `tests/a/b`

Reply:
added